README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolkit** for defining, building and generating **APIs, documentation and SDKs**. Bring an existing OpenAPI definition or describe your endpoints in a Fern specification. Fern generates backend code, specifications in various formats, and client libraries with a single command.
```

Comment:
this leaves me thinking: which ones?

Reply:
I liked the original paragraphs better than what we have now: 

> Fern is an open source toolset for defining, building and generating APIs and SDKs. Fern can be used in your existing projects through OpenAPI, and integrate effortlessly into your existing workflows. It can also serve as the basis of your API development, by describing your endpoints in a Fern specification, and having Fern generate your API backend code for you.

> This repository contains the Fern compiler. The compiler transforms an API definition, either in Fern or OpenAPI format into various code artifacts, such as Client Libraries, SDKs, or even fully-fledged Express or FastAPI backends.

> In short, Fern takes care of the boilerplate associated with developing APIs, so you can focus on delivering business value.

Reply:
What do you think about starting with a sentence like this: 
> Fern is an open source toolset for designing, building and consuming REST APIs. 

Here's how i think about the 3 phases: 

**Design** Developers can opt to design their APIs in the Fern Specification. If they'd rather use OpenAPI though, we support reading that as well. 

**Build** Fern generates boilerplate code for backends written in FastAPI, Express, and Spring Boot. Fern generates all the networking logic and provides type safety, so that developers can focus on the business logic. 

**Consume** Fern generates client libraries for multiple programming languages. We've built the client library generators from the ground up so that they follow the idioms of each language and feel handwritten.